third_party/blink/renderer/core/url/dom_url.cc

```diff
@@ -95,12 +95,7 @@ String DOMURL::CreatePublicURL(ExecutionContext* execution_context,
 
 URLSearchParams* DOMURL::searchParams() {
   if (!search_params_) {
-    if (recordreplay::IsRecordingOrReplaying() && !recordreplay::AreAssertsDisabled()) {
-      std::string stack;
-      recordreplay::GetCurrentJSStack(&stack);
-      recordreplay::Assert("[RUN-2324-2325] DOMURL::searchParams %s stack=%s",
-                           Url().GetString().Utf8().c_str(), stack.c_str());
-    }
+    recordreplay::AutoDisallowEvents disallowed("DOMURL::searchParams");
```

Comment:
Add a comment or issue reference within the string?

Reply:
yeah, good call.  adding.